src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -372,6 +375,13 @@ export class StudyViewPageStore {
                 return acc;
             }, [] as CopyNumberGeneFilter[]);
         }
+        this.defaultFilters = filters;
+    }
+
+    @computed
+    get isInitialLoading(): boolean {
+        // TODO: figure out why isEqual cannot be applied to these two observables
+        return _.isEqual(JSON.parse(JSON.stringify(this.defaultFilters)), JSON.parse(JSON.stringify(this.filters)));
```

Comment:
should we rename defaultFilters = initialFilters 
and filter = currentFilters?

Reply:
I changed the defaultFilters to initialFilters. Not the filters yet. It will create too many conflicts with my other pr. I will do a cleanup in a separate pr. Will follow up when working on this issue https://github.com/cBioPortal/cbioportal/issues/5197